content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -3327,3 +3327,38 @@ model User {
 
 </tab>
 </TabbedContent>
+
+## <inlinecode>type</inlinecode>
+
+<Admonition type="warning">
+
+Composite types are available **for MongoDB only** in versions 3.12.0 and later, and in versions 3.10.0 and later if you enable the `mongodb` Preview feature flag.
```

Comment:
It might be because I'm reading this at 9am, but I had to read this a couple of times to parse it. Do we mean that 
* composite type are available for MongoDB only or
* in version 3.12.0 and later, composite types are available for MongoDB only? But they're available for all database types in earlier versions?

I think the two criteria - version, and being MongoDB only - would be clearer if they were separated out.

Reply:
Thanks, separated them out now